### ISSUE TITLE
Portals unit test failure

### ISSUE THREAD

**Issue by jdinan** [Sandia-OpenSHMEM/SOS#574]
I'm seeing an odd unit test failure in the Portals build:
```
$ mpiexec -np 2 -hosts compute-0-1,compute-0-2 test/unit/shmem_ctx
[0000] ERROR: ../../src/shmem_synchronization.h:30 shmem_internal_quiet, return code -1
17:31:52 ~/projects/sandia-shmem/build

$ mpiexec -np 2 -hosts compute-0-1,compute-0-2 test/unit/shmem_ctx
Error: total_done is 2063, expected 2048

$ mpiexec -np 2 -hosts compute-0-1,compute-0-2 test/unit/shmem_ctx
Error: total_done is 2074, expected 2048
[0001] WARN:  ../../src/transport_portals4.c:675: shmem_transport_fini
[0001]        get count mismatch: 1384, 1387
[0000] WARN:  ../../src/transport_portals4.c:675: shmem_transport_fini
[0000]        get count mismatch: 877, 879
```

Here's the config:
```
../configure --prefix=/home/dinanjam/opt/sos-portals --with-portals4=/home/dinanjam/opt/portals4 --enable-pmi-simple --disable-cxx --enable-debug --enable-picky --enable-error-checking --enable-remote-virtual-addressing
```

**Comment by jdinan** [comment]
Might be caused by my Portals build; looks like it's a ways behind the current master.  Might also be cause by ```--enable-shmem``` in the Portals config.  I'll try again..

**Comment by jdinan** [comment]
Closed by #575
